recipes/hg-evolve/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "hg-evolve" %}
+{% set version = "8.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c7ef633c1ab9ef4ef9da8c74165a7b12daa32a38d6b3de9324ba4ce2b42ea126
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python < 3
```

This has to be formatted like this else some parts of conda-forge have issues rendering the recipe:
```suggestion
    - python <3
```
(one space before the operator, no spaces after)